app/components/admin/design_header_component.html.erb

```diff
@@ -27,7 +27,7 @@ See COPYRIGHT and LICENSE files for more details.
 
 ++#%>
 
-<% helpers.html_title t(:label_administration), t(:label_custom_style), @title %>
+<% helpers.html_title t(:label_administration), t(:label_custom_style), selected_tab(@tabs)[:label] %>
```

Comment:
No issue, but an observation: when `selected_tab` cannot find the currently selected tab, it will [return the first entry of the given Array as fallback](https://github.com/opf/openproject/blob/488e7ede14d4967fe74184a289113e1087832604/app/helpers/tabs_helper.rb#L41). In other words: `@tabs.first` is the fallback if no currently selected tab can be found.

Furthermore, if `@tabs` _was_ an empty Array, `@tabs.first` would evaluate to `nil`. And `nil[:label]` would raise an Error.

Luckily `@tabs` is hard-coded to be a list with three entries (see [CustomStylesHelper#design_tabs](https://github.com/opf/openproject/blob/488e7ede14d4967fe74184a289113e1087832604/app/helpers/custom_styles_helper.rb#L30). It is very unlikely that we remove the tabs in the future and keep the selected_tabs helper here at the same time. So we are fine without adding hard-to-read guards here 😀

Reply:
Hi @EinLama thanks for explaining 👍